src/Microsoft.Azure.WebJobs.Extensions.ServiceBus/Listeners/ServiceBusListener.cs

```diff
@@ -177,5 +209,338 @@ private void ThrowIfDisposed()
                 throw new ObjectDisposedException(null);
             }
         }
+
+        private void InitializeMetricsClients()
+        {
+            if (_metricsReceiver == null)
+            {
+                _metricsReceiver = _messagingProvider.CreateMessageReceiver(_entityPath, _serviceBusAccount.ConnectionString);
+            }
+
+            if (_managementClient == null)
+            {
+                _managementClient = new ManagementClient(_serviceBusAccount.ConnectionString);
+            }
+        }
+
+        async Task<ScaleMetrics> IScaleMonitor.GetMetricsAsync()
```

Comment:
`async Task<ScaleMetrics> IScaleMonitor.GetMetricsAsync()` and public `async Task<ServiceBusTriggerMetrics> GetMetricsAsync()` have the same body. Can we create a private method:
`return _entityType == EntityType.Queue ? await GetQueueMetricsAsync() : await GetTopicMetricsAsync();`